src/shared/sleep-config.c

```diff
@@ -435,5 +419,65 @@ static int can_sleep_internal(const char *verb, bool check_allowed) {
 }
 
 int can_sleep(const char *verb) {
-        return can_sleep_internal(verb, true);
+        int r;
+        _cleanup_(free_sleep_configp) SleepConfig *sleep_config = NULL;
```

Comment:
Not that it matter too much, but we usually put `int r` as the last definition. Please switch the order for consistency here.